data/events.json

```diff
@@ -822,5 +822,141 @@
     "end": "2023-04-19 23:59",
     "groupId": 10,
     "filterId": 5
+  },
+  {
+    "title": "[CESIUM] CTF's",
+    "place": "Room 1.09 CP7",
```

Comment:
Please update this to: `Ed. <nr> - <room>`, like so: `Ed. 7 - 1.09`. If the building is 1, 2 or 3 use `CP<nr>` instead of `Ed. <nr>`. This is so it matches the format used on the Schedule page ;)